GraphicLibrary/Src/GraphicLib.hpp

```diff
@@ -61,4 +62,30 @@ class GraphicLib : public IGraphic {
          * @return KEY_MAP The key pressed.
          */
         std::size_t getKeyInput();
+
+        /**
+         * @brief Draw an OBJ model on the screen.
+         *
+         * @param objPath Path to the OBJ Model to draw.
+         */
+        void drawOBJ(std::string objPath, float posx, float posy, float posz, float scale);
+
+        /**
+         * @brief Start to draw on the window.
+         * Must be call at the begin of the Game loop.
+         *
+         */
+        void startDraw();
+
+        /**
+         * @brief End to draw on the window.
+         * Must be call at the end of the Game loop.
+         *
+         */
+        void endDraw();
+
+    private:
+
+        Camera                                  _camera;    //< Player camera.
```

Comment:
Camera ?

Reply:
Camera is a raylib structure to handle camera.

Reply:
Yes but here you have nom variable name, its normal ?